File: Corredor/Digitales/Java/src/Escuela.java
import java.util.ArrayList;

public class Escuela {
    private String nombre;
    private String direccion;
    private ArrayList<Profesor> profesores;
    private ArrayList<Estudiante> estudiantes;

    public Escuela(String nombre, String direccion) {
        this.nombre = nombre;
        this.direccion = direccion;
        this.profesores = new ArrayList<>();
        this.estudiantes = new ArrayList<>();
    }

    public void agregarProfesor(Profesor p) {
        profesores.add(p);
    }

    public void agregarEstudiante(Estudiante e) {
        estudiantes.add(e);
    }

    public void mostrarInfo() {
        System.out.println("🎵 Escuela: " + nombre + " | Dirección: " + direccion);
        System.out.println("Profesores: " + profesores.size() + " | Estudiantes: " + estudiantes.size());
    }
}

File: Corredor/Digitales/JavaScript/Escuela.js
class Escuela {
    constructor(nombre, direccion) {
        this.nombre = nombre;
        this.direccion = direccion;
        this.profesores = [];
        this.estudiantes = [];
    }

    agregarProfesor(profesor) {
        this.profesores.push(profesor);
    }

    agregarEstudiante(estudiante) {
        this.estudiantes.push(estudiante);
    }

    mostrarInfo() {
        console.log(`🎵 Escuela: ${this.nombre} | Dirección: ${this.direccion}`);
        console.log(`Profesores: ${this.profesores.length} | Estudiantes: ${this.estudiantes.length}`);
    }
}

module.exports = Escuela;